src/components/views/rooms/RoomList.js

```diff
@@ -255,6 +256,14 @@ module.exports = React.createClass({
         }
     },
 
+    onMouseEnter: function(ev) {
+        this.setState({hover: true});
+    },
+
+    onMouseLeave: function(ev) {
+        this.setState({hover: false});
```

Comment:
Should we trigger a list refresh at this point to account for any room list changes that may have occurred while hovering but were ignored and so have not yet been displayed? Perhaps add `this._delayedRefreshRoomList();` after this line.